pipelines/disturbance/validate_zonal_statistics.py

```diff
@@ -1,6 +1,108 @@
+    'WSM', 'SMR', 'STP', 'SAU', 'SEN', 'SRB', 'SYC', 'SLE', 'SGP', 'SXM', 'SVK', 'SVN', 'SLB', 'SOM', 'ZAF', 'SGS',
+    'SSD', 'ESP', 'LKA', 'SDN', 'SUR', 'SJM', 'SWE', 'CHE', 'SYR', 'TWN', 'TJK', 'TZA', 'THA', 'TLS', 'TGO', 'TKL',
+    'TON', 'TTO', 'TUN', 'TUR', 'TKM', 'TCA', 'TUV', 'UGA', 'UKR', 'ARE', 'GBR', 'USA', 'UMI', 'URY', 'UZB', 'VUT',
+    'VEN', 'VNM', 'VGB', 'VIR', 'WLF', 'ESH', 'YEM', 'ZMB', 'ZWE'
+]
+
+sbtn_natural_lands_classes = [
+    "Forest", "Short vegetation", "Water", "Mangroves", "Bare", "Snow/Ice", "Wetland forest", "Peat forest",
+    "Wetland short vegetation", "Peat short vegetation",  "Cropland", "Built-up", "Tree cover", "Short vegetation",
+    "Water", "Wetland tree cover", "Peat tree cover", "Wetland short vegetation", "Peat short vegetation", "Bare"
+]
+
+class DistZonalStats(pa.DataFrameModel):
+    country: Series[int] = pa.Field(eq=76) # gadm id for Brazil
+    region: Series[int] = pa.Field(eq=20) # gadm id for adm1 AOI
+    subregion: Series[int] = pa.Field(lt=170) # placeholder adm2
+    alert_date: Series[int] = pa.Field(ge=731, le=1640) # julian date between 2023-01-01 to latest version
+    confidence: Series[int] = pa.Field(ge=2, le=3) # low confidence, high confidence
+    value: Series[int]
+    
+    class Config:
+        coerce = True
+        strict = True
+        name = "ZonalStatsSchema"
+        ordered = True
+        unique_columns = ["country", "region", "subregion", "alert_date", "confidence"]
+
+    @staticmethod
+    def calculate_alert_counts(df: pd.DataFrame) -> dict:
+        """Calculate the number of alerts by confidence level."""
+        alert_counts = df["confidence"].value_counts().to_dict()
+        return {
+            "low_confidence": alert_counts.get(2, 0),
+            "high_confidence": alert_counts.get(3, 0),
+        }
+    
+    @staticmethod
+    def spot_check_julian_dates(df: pd.DataFrame, julian_dates: List[int]) -> pd.DataFrame:
+        filtered_by_date_df = df[df["alert_date"].isin(julian_dates)]
+        filtered_by_date_df = filtered_by_date_df.sort_values(by="alert_date").reset_index(drop=True)
+        return filtered_by_date_df[["alert_date", "confidence", "value"]]
+    
+class NaturalLandsZonalStats(pa.DataFrameModel):
+    countries: Series[str] = pa.Field(isin=isos)
+    regions: Series[int] = pa.Field()
+    subregions: Series[int] = pa.Field()
+    natural_lands: Series[str] = pa.Field(isin=sbtn_natural_lands_classes)
+    area: Series[float] = pa.Field(ge=0)
+    
+def validates_zonal_statistics(parquet_uri: str) -> bool:
     """Validate Zarr to confirm there's no issues with the input transformation."""
-    # check global stats identical zarr/tile stats
-    # check diff threshold for AOIs
-    # double check against alternative method
-    pass
+    
+    logger = get_run_logger()
+
+    # load local results
+    validation_df = pd.read_csv("../notebooks/validation_stats.csv")
+    logger.info("Loaded validation stats.")
```

Comment:
@manukala6 it seems like you're creating the stats in a notebook and then checking it here. But I think the intention was to actually run the alternative stats method here, and then compare it to the parquet.